data/base-theurgy.yaml

```diff
@@ -1,4 +1,48 @@
 ---
+immortal_to_aspect:
+  # Neutral
+  Everild: boar
+  Kertigen: raven
+  Damaris: panther
+  Tamsine: cat
+  Meraud: wolf
+  Truffenyi: ox
+  Glythtide: ram
+  Chadatru: lion
+  Faenella: wren
+  Hodierna: unicorn
+  Eluned: dolphin
+  "Hav'roth": cobra
```

Comment:
I don't think you need the `"` around this. Have you tried without?

Reply:
Done; haven't tested the change yet, though.